tests/subsys/storage/flash_buffered_write/prj.conf

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ZTEST=y
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
```

Comment:
Seems like it would be better to have an overlay with `CONFIG_MPU_ALLOW_FLASH_WRITE=y` for the platforms which depends on it.

Reply:
I agree, will change the logics